src/compiler/msvc.rs

```diff
@@ -261,6 +262,7 @@ msvc_args!(static ARGS: [ArgInfo<ArgData>; _] = [
     msvc_flag!("Zi", DebugInfo),
     msvc_flag!("c", DoCompilation),
     msvc_take_arg!("deps", PathBuf, Concatenated, DepFile),
+    msvc_take_arg!("external:I", PathBuf, Separated, ExternalIncludePath),
```

Comment:
I wonder if it /has/ to be separated or if it's actually something that can also be concatenated, like normal `-I`.

Reply:
I think it has to be separated -- at least the blogpost points that direction, and I would think that requiring it to be separated also eliminates parsing ambiguities if there was some future `/external:Ifoo` flag.  But I have not actually experimented with this to find out.

Reply:
Trying out with MSVC 2019, it seems `-external:Ifoo` is valid.